itkufs/reports/forms.py

```diff
@@ -99,7 +99,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         self.fields["extra_accounts"].choices = [
-            (a.id, a.name) for a in group.account_set.all()
+            (a.id, a.name) for a in group.account_set.all() if a.active
```

Comment:
Or to do it in the DB instead of Python :-)

```suggestion
            (a.id, a.name) for a in group.account_set.filter(active=True)
```